build.gradle

```diff
@@ -0,0 +1,44 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
+}
+
+group 'com.google.cloud.dataproc.jdbc'
+version '1.0-SNAPSHOT'
+
+// Dataproc (and the Hadoop ecosystem in general) still use Java 8
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    implementation 'com.google.guava:guava:29.0-jre'
+    testCompile group: 'junit', name: 'junit', version: '4.12'
```

Comment:
JUnit 4.13 is available, but for a new projects we should use JUnit 5, if possible.

Reply:
Sure, thanks for pointing it out.